libs/cli/setup.go

```diff
@@ -92,7 +92,7 @@ func (e Executor) Execute() error {
 			const size = 64 << 10
 			buf := make([]byte, size)
 			buf = buf[:runtime.Stack(buf, false)]
-			fmt.Fprintf(os.Stderr, "ERROR: %v\n%s\n", err, buf)
+			Stderr.Fprintf(os.Stderr, "ERROR: %v\n%s\n", err, buf)
```

Comment:
This isn't a declared variable.

@bhaskarvilles, what problem is it that you would like to solve here?

Reply:
The error message from `fmt` will expose sensitive information from `os.Stderr`, it was a good practice to use `Stderr`